slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/session/mapper/SessionMapper.java

```diff
@@ -0,0 +1,15 @@
+package com.binarystudio.academy.slidez.domain.session.mapper;
+
+import com.binarystudio.academy.slidez.domain.session.dto.SessionUpdateDto;
+import com.binarystudio.academy.slidez.domain.session.model.Session;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring")
+public interface SessionMapper {
+
+	SessionMapper INSTANCE = Mappers.getMapper(SessionMapper.class);
+
+	Session mapSesionUpdateDtoToSession(SessionUpdateDto sessionUpdateDto);
+
```

Comment:
You missed a second letter 's' in mapSessionUpdateDtoToSession

Reply:
Fixed